zerver/views/realm_export.py

```diff
@@ -45,3 +48,17 @@ def export_realm(request: HttpRequest, user: UserProfile) -> HttpResponse:
 def get_realm_exports(request: HttpRequest, user: UserProfile) -> HttpResponse:
     realm_exports = get_realm_exports_serialized(user)
     return json_success({"exports": realm_exports})
+
+@require_realm_admin
+def delete_realm_export(request: HttpRequest, user: UserProfile, export_id: int) -> HttpResponse:
+    try:
+        export = RealmAuditLog.objects.get(id=export_id,
+                                           realm=user.realm,
+                                           event_type="realm_exported")
+    except RealmAuditLog.DoesNotExist:
+        return json_error(_("The export doesn't exist"))
+
+    if ujson.loads(export.extra_data).get('deleted_timestamp') is not None:
+        return json_error(_("Export already deleted"))
```

Comment:
Split the JSON load on a separate line, and do an `if 'deleted_timestamp' in export_data`.

Reply:
Done.